docs/content/services/connect_mongodb_database_cluster.md

```diff
@@ -83,125 +81,177 @@ Now you are going to set up a MongoDB cluster to connect to your local applicati
 
-- Now you can simply copy the URI by clicking on the **Copy** icon. Before pasting the URI in your application code, make sure to replace **<password>** in the URI with the actual password you provided for the relevant user created earlier. 
+-   Now you can simply copy the URI by clicking on the **Copy** icon. Before pasting the URI in your application code, make sure to replace **<password>** in the URI with the actual password you provided for the relevant user created earlier.
 
 <figure markdown>
   <kbd>![32.png](../../img/services/connect_mongodb_database_cluster/32.png){width="600"}</kbd>
-</figure> 
\ No newline at end of file
+</figure>
+
+## MongoDB Compass (Optional)
+
+MongoDB Compass is a GUI tool for MongoDB. It is a cross-platform application that allows you to connect to your MongoDB database and perform various operations on it.
+We recommend you to use MongoDB Compass to access your database cluster during the development phase of the project.
+
+-   _Expected benifits of using MongoDB Compass are as follows._
+
+    -   You can save time by using the GUI tool.
+    -   You can view the data in your database in a more readable format.
+    -   You can perform CRUD operations on your database.
+    -   You can view the structure of your database.
+    -   You can view the logs of your database.
+
+-   _How to get MongoDB Compass?_
+
+    -   You can download MongoDB Compass from [here](https://www.mongodb.com/try/download/compass).
+    <figure markdown>
+      <kbd>![compass-download.png](../../img/services/connect_mongodb_database_cluster/compass-download.png){width="600"}</kbd>
+    </figure>
+    <sub>Select the appropriate latest stable version for your operating system and download it.</sub>
+
+    -   After downloading, run the downloaded file and install MongoDB Compass. (You can follow the installation steps [here](https://docs.mongodb.com/compass/master/install/).)
+    <figure markdown>
+      <kbd>![compass-installing.png](../../img/services/connect_mongodb_database_cluster/compass-installing.png){width="600"}</kbd>
+    </figure>
+    <sub>After running the downloaded file, you will see the installation wizard. Wait until the installation is completed.</sub>
+
+    -   After the installation is completed, it will automatically open the MongoDB Compass. If not, you can open it from the start menu. You can see the following screen.
+    <figure markdown>
+      <kbd>![compass-first-view.png](../../img/services/connect_mongodb_database_cluster/compass-first-view.png)</kbd>
+    </figure>
+        Close the welcome screen and click on **New connection**.
+
+    -   Then you need to provide the connection details. You can find the connection details in the **Connect** tab of your database cluster. You can see the following screen. Copy the connection URI and paste it in the **Connection String Only** field in MongoDB Compass. Then click on **Save and Connect**.
```

Comment:
- How can I **find** the connection details in the **Connect** tab?
- Where's the **Connection String Only** field?
- It's better to point out the UI elements in the images which will be used by the user.